diff --git a/dataset.py b/dataset.py
index 62cff7d..1601390 100644
--- a/dataset.py
+++ b/dataset.py
@@ -11,6 +11,7 @@ class ModelNetDataLoader(Dataset):
         self.root = root
         self.npoints = npoint
         self.uniform = uniform
+        print(self.root)
         self.catfile = os.path.join(self.root, 'modelnet40_shape_names.txt')
 
         self.cat = [line.rstrip() for line in open(self.catfile)]
diff --git a/train_partseg.py b/train_partseg.py
index 7100699..2c2f27c 100644
--- a/train_partseg.py
+++ b/train_partseg.py
@@ -42,7 +42,8 @@ def to_categorical(y, num_classes):
         return new_y.cuda()
     return new_y
 
-@hydra.main(config_path='config', config_name='partseg')
+@hydra.main(config_path='config', config_name='partseg', version_base='1.2')
+
 def main(args):
     omegaconf.OmegaConf.set_struct(args, False)
 
@@ -64,6 +65,8 @@ def main(args):
     num_part = args.num_class
     shutil.copy(hydra.utils.to_absolute_path('models/{}/model.py'.format(args.model.name)), '.')
 
+    if not torch.cuda.is_available():
+        print("OMG CUDA ARRETE DE FAIRE CHIER!!")
     classifier = getattr(importlib.import_module('models.{}.model'.format(args.model.name)), 'PointTransformerSeg')(args).cuda()
     criterion = torch.nn.CrossEntropyLoss()
 
